config/StoredProcedures/EditUserByUsername

```diff
@@ -1,14 +0,0 @@
-DROP PROCEDURE IF EXISTS EditUserByUsername;
-
-DELIMITER $$
-CREATE PROCEDURE EditUserByUsername
-(IN _username varchar(255), IN _firstname varchar(255), IN _lastname varchar(255), IN _permissions int, IN _confirmation bit)
-
-BEGIN
-
-UPDATE Users
-SET FirstName = _firstname, LastName = _lastname, PermsID = _permissions, IsConfirmed = _confirmation
-WHERE Username = _username;
-END $$
-
-DELIMITER ;
```

Comment:
Ah, I think the case you're accounting for is when we want to update a user but don't know up front what the confirmation bit should be set to, so we just want to leave it as is.